static/src/stylesheets/module/content/_gallery.scss

```diff
@@ -245,3 +245,333 @@
+    }
+
+    .content__main-column {
+        max-width: none;
+
+        @include mq(tablet) {
+            margin-left: gs-span(3);
+        }
+    }
+
+    @include mq(tablet) {
+        .content__dateline,
+        .content__labels,
+        .byline,
+        .content__meta-container,
+        .meta__break,
+        .meta__save-for-later,
+        .content__headline,
+        .content__standfirst {
+            margin-left: gs-span(3);
+            margin-right: gs-span(1);
+            max-width: gs-span(8);
+        }
+
+        .byline, .content__dateline {
+            border: 0;
+        }
+
+        .content__standfirst {
+            padding-bottom: 0;
+            padding-top: 0;
+
+            &:before {
+                content: none;
+            }
+        }
+    }
+
+    // overriding defaults
+    .meta__save-for-later {
+        position: relative;
+        left: 0;
+    }
+
+    // TODO: Replace all media tones instead of overwriting the defaults
+    &.tonal--tone-media {
+
+        // Overwriting media tone header so that you can see the image instead
+        .tonal__header {
+            background-color: rgba(0, 0, 0, 0);
+        }
+
+        .meta__numbers {
+            color: $neutral-3;
+        }
+
+        .content__dateline,
+        .sharecount__heading {
+            color: $neutral-2;
+        }
+
+        .byline a,
+        .sharecount__value {
+            color: $neutral-3;
+        }
+
+        .save-for-later__button--save .inline-icon {
+            background-color: $multimedia-support-5;
+        }
+    }
+
+    .content__wrapper--headline {
+        background-color: rgba(0, 0, 0, .6);
+        background: linear-gradient(rgba(0, 0, 0, .5), $multimedia-support-5);
+        position: absolute;
+        left: 0;
+        right: 0;
+        bottom: 0;
+    }
+
+    .content__headline {
+        &:before {
+            content: none;
+        }
+    }
+
+    .inline-camera svg {
+        width: 32px;
+        height: auto;
+    }
+
+    .content__standfirst {
+        color: $neutral-5;
+    }
+
+    .content__meta-container {
+        border: 0;
+        position: relative;
+        width: auto;
+        margin-bottom: 0;
+    }
+
+    .meta__social {
+        border-top: 1px dotted $neutral-2;
+        border-bottom: 1px dotted $neutral-2;
+    }
+
+    .meta__break {
+        border-top: 1px dotted $neutral-2;
+        display: block;
+        width: 10%;
+    }
+
+    .meta__numbers {
+        border: 0;
+        position: absolute;
+        right: 0;
+        top: 0;
+
+        .sharecount__heading {
+            padding-left: 0;
+        }
+    }
+
+    .content__series-label {
+        padding-top: 0;
+        margin-left: 1rem;
+    }
+
+    .content__series-label,
+    .content__section-label {
+        display: inline-block;
+    }
+
+    .content__labels {
+        border: 0;
+        margin-bottom: 0;
+        width: auto;
+        float: none;
```

Comment:
Instead of overriding these styles, perhaps we should refactor `content__labels` so it only has common styles and then create another class such as `content__labels--article` with article specific styles. This way we won't need to have any overrides. What do you think?

Overrides lead to lots of mess in CSS :-(